stylesheets/xslt/tei.p5.xsl

```diff
@@ -370,41 +496,7 @@
           </span>
         </span>
         
-       <!-- 
-        <span class="hr">&#160;</span>
-        <span class="pageimage">
-          <a>
-            <xsl:attribute name="href">
-              <xsl:value-of select="$fileroot"/>
-              <xsl:text>figures/800/</xsl:text>
-              <xsl:value-of select="graphic/@facs"/>
-              <xsl:text>.jpg</xsl:text>
-            </xsl:attribute>
-            <xsl:attribute name="rel">
-              <xsl:text>prettyPhoto[pp_gal]</xsl:text>
-            </xsl:attribute>
-            <xsl:attribute name="title">
-              <xsl:text>&lt;a href="</xsl:text>
-              <xsl:value-of select="$fileroot"/>
-              <xsl:text>figures/800/</xsl:text>
-              <xsl:value-of select="@facs"/>
-              <xsl:text>.jpg</xsl:text>
-              <xsl:text>" target="_blank" &gt;open image in new window&lt;/a&gt;</xsl:text>
-            </xsl:attribute>
-            <img>
-              <xsl:attribute name="src">
-                <xsl:value-of select="$fileroot"/>
-                <xsl:text>figures/250/</xsl:text>
-                <xsl:value-of select="@facs"/>
-                <xsl:text>.jpg</xsl:text>
-              </xsl:attribute>
-            </img>
-          </a>
-        </span>-->
-        
-        
-        
-        
+      

```

Comment:
It would definitely be nice to get rid of this chunk if the other code is working, or maybe put it in a gist or something?  Otherwise it may be there forever, potentially, haha.